src/components/MapboxMap.tsx

```diff
@@ -161,10 +172,79 @@ export function MapboxMap<T>({
   }, [PinComponent, getCoordinate, locationResults]);
 
   return (
-    <div ref={mapContainer} className='h-full w-full' />
+    <div className='grid h-full w-full' ref={parentContainer}>
+      <div ref={mapContainer} className="col-span-full row-span-full invisible"/>
+      {mapboxStaticImageDiv}
+    </div>
   );
 }
 
+function useMapboxStaticImage(
```

Comment:
Instead of a hook that returns JSX would it make sense for this to just be a component?
It might make it easier to add a story for it as well then?

Reply:
sure!